master/types.html

```diff
@@ -901,8 +901,8 @@ <h3 id="InterfaceSVGGeometryElement">Interface SVGGeometryElement</h3>
 If either of the <var>x</var> or <var>y</var> properties on <var>point</var> are infinite or NaN,
 then the method must return false.</p>
 
-<p class="note"><a>isPointInStroke</a> is aligned
-with the <a>isPointInPath</a> method on the <a>CanvasDrawPath</a> mixin as much as the SVGcontext
+<p class="note"><a href="types.html#__svg__SVGGeometryElement__isPointInStroke">isPointInStroke</a> is aligned
+with the <a>isPointInPath</a> method on the <a>CanvasDrawPath</a> mixin as much as the SVG context
```

Comment:
`isPointInFill` links to the correct section. Checked it again. You are right about `isPointInStroke ` though. I decided to keep `isPointInStroke` linking to the HTML Canvas spec since this link is more likely to update. Or requires updates... not sure how stable HTML spec links are.

Reply:
Well, stable or not, it is the only up-to-date Canvas spec. The W3C version hasn't been republished in years.

Reply:
Whoops! It looks like you merged without actually adding this change. The "suggested changes" is just that -- you need to accept the suggestion for it to affect the PR.

Reply:
@AmeliaBR Little confused by the above comments. In `definitions.xml` we reference the WHATWG spec and not W3C which is more likely to change. As said, I checked all links related to `isPointInFill`, `isPointInStroke` or `isPointInPath`. As far as I can tell, the links do reference the HTML Canvas spec where intended and the SVG versions in other cases.
I probably just misunderstood your change request though.

Reply:
Ah, My suggested change wasn't which spec, it was which method!  The note about stroke was linking to the method for fill/path.

Anyway, I'll make the change & commit directly.